api/tests/src/api/extracts_v1/test_extracts_routes.py

```diff
@@ -0,0 +1,127 @@
+from datetime import datetime, timedelta
+
+import pytest
+
+from src.constants.lookup_constants import ExtractType
+from src.db.models.extract_models import ExtractMetadata
+from tests.src.db.models.factories import ExtractMetadataFactory
+
+
+@pytest.fixture(autouse=True)
+def clear_extracts(db_session):
+    db_session.query(ExtractMetadata).delete()
+    db_session.commit()
+    yield
+
+
+def test_extract_metadata_get_default_dates(
+    client, api_auth_token, enable_factory_create, db_session
+):
+    """Test that default date range (last 7 days) is applied when no dates provided"""
+    ExtractMetadataFactory.create_batch(
+        2,
+        extract_type=ExtractType.OPPORTUNITIES_XML,
```

Comment:
Not specifically for this PR, but I missed it when you added extract type - we want JSON, not XML for this file type.

With how the lookup values work, it should be safe to just change this (DB migrations will resync the value to the DB - and nothing is using it in the API yet).

Reply:
Yep makes sense! I was just making this change when adding the db record when the task runs.